src/NewFamilyMember.scss

```diff
@@ -0,0 +1,186 @@
+#root {
+  // max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+div.Header {
```

Comment:
I'd recommend only using `class names` for styling. There is no need to use a combination of tag + classname. I would also recommend against using ids for styling

More importantly, it's also really common to write class names in [kebab-case](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841)